modules/food-menu/fancy-menu.js

```diff
@@ -94,6 +96,13 @@ export class FancyMenu extends React.Component<Props, State> {
 		})
 	}
 
+	onPressRow(item: MenuItem, icons: MasterCorIconMapType) {
+		this.props.navigation.navigate('MenuItemDetailView', {
+			item,
+			icons,
+		})
```

Comment:
Can you put these on one line, please? `{item, icons}` (unless Prettier is forcing the wrapping when you inline them)